process/sp.py

```diff
@@ -277,14 +277,15 @@
         population_density,
         intersection_density,
     )
-        
+
     samplePointsData = samplePointsData[["hex_id", "edge_ogc_fid", "geometry"]].join(full_nodes, how="left")
-    
+
     # create binary distances evaluated against accessibility distance
+    # Options for distance decay accessibility scores are available in setup_sp.py module
     binary_names = [f"{x.replace('nearest_node','access')}_binary" for x in distance_names]

```

Comment:
We should change the 'binary_names' variable and suffix to something like 'access_score' so it makes sense in a generalised context where a user can configure usage of a custom function